HRIS.Services/Interfaces/ITerminationService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using HRIS.Models;
+
+namespace HRIS.Services.Interfaces;
+
+public interface ITerminationService
+{
+    /// <summary>
+    ///     Check if user exist
+    /// </summary>
+    /// <param name="email"></param>
+    /// <returns></returns>
```

Comment:
The CheckTerminationExistst return of type bool but the returns in the comment above states nothing is returned
```suggestion
    /// <returns>Bool on whether the termination exists</returns>
```